samples/sensor/ism330dhcx/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  name: ISM330DHCX accelerometer and gyrometer sensor
+tests:
+  sample.sensor.ism330dhcx:
+    harness: console
+    tags: sensors
+    depends_on: ism330dhcx
```

Comment:
please use `filter: dt_compat_enabled("your-sensor-compatible")`

Reply:
Thanks for the review. The patch has been updated to use the suggested filter.